reference/configuration/framework.rst

```diff
@@ -169,39 +293,431 @@ see :doc:`/cookbook/request/load_balancer_reverse_proxy`.
 
         # app/config/config.yml
         framework:
-            trusted_proxies:  [192.0.0.1, 10.0.0.0/8]
+            trusted_hosts:  ['acme.com', 'acme.org']
+
+    .. code-block:: xml
+
+        <!-- app/config/config.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <trusted-host>acme.com</trusted-host>
+                <trusted-host>acme.org</trusted-host>
+                <!-- ... -->
+            </framework>
+        </container>
+
+    .. code-block:: php
+
+        // app/config/config.php
+        $container->loadFromExtension('framework', array(
+            'trusted_hosts' => array('acme.com', 'acme.org'),
+        ));
+
+Hosts can also be configured using regular expressions, which make it easier to
+respond to any subdomain:
```

Comment:
I would almost rather save us some space with code blocks and include the examples inline in the sentence:

```rst
Hosts can also be configured using regular expressions (e.g. ``.*\.?acme.com$`` or ``.*\.?acme.org$``), which make it ...
```